site/index.html

```diff
@@ -85,6 +83,16 @@ <h1 class="component__title">
       ga('send', 'pageview');
 
       new window.Blazy();
+
+      var componentList = document.querySelectorAll('.component');
+      function search(s) {
+        var text = ('innerText' in document.body)? 'innerText' : 'textContent';
+        for (var i = 0; i < componentList.length; i++) {
+          componentList[i].style.display = (componentList[i][text].indexOf(s) !== -1) ? 'flex' : 'none';
```

Comment:
would say `... ? null : 'none'`, as `null` will remove inline styles and revert to whatever is in the CSS.